var/spack/repos/builtin/packages/sentieon-genomics/package.py

```diff
@@ -15,17 +13,13 @@ class SentieonGenomics(Package):
     Please set the path to the sentieon license server with:
 
     export SENTIEON_LICENSE=[FQDN]:[PORT]
-
-    Note: A manual download is required.
-    Spack will search your current directory for the download file.
-    Alternatively, add this file to a mirror so that Spack can find it.
-    For instructions on how to set up a mirror, see
-    https://spack.readthedocs.io/en/latest/mirrors.html"""
+    """
 
     homepage = "https://www.sentieon.com/"
-    url      = "file://{0}/sentieon-genomics-201808.01.tar.gz".format(os.getcwd())
-    manual_download = True
+    url      = "https://s3.amazonaws.com/sentieon-release/software/sentieon-genomics-201808.01.tar.gz"
```

Comment:
@snehring Thanks, can you add a comment on top of the `url` saying we are using the permalink published in their Dockerfile and adding the link above?

@donFreed I assume the answer is yes, but just to double check, would it be fine to redistribute the binaries from:
```console
https://s3.amazonaws.com/sentieon-release/software/sentieon-genomics-201808.01.tar.gz
```
which was extracted from your public Dockerfile?

Reply:
Yes, redistribution of the precompiled binaries is OK.
Just be aware that we are a commercial software company and the software requires an active software license before it can be used.

Reply:
Thanks all for the clarifications.

Reply:
Here is the text of our EULA, https://www.sentieon.com/EULA/eula-aws.html. It would be great if this could somehow be pointed to in spack.

The EULA expressly forbids copying without prior approval. Hopefully the above comment is sufficient?

Reply:
@tgamblin would you like me to include a link to the EULA or revert to manual download?

Reply:
I've pushed a commit that adds a link to the eula. @tgamblin let me know if this is satisfactory.